mongodbatlas/resource_mongodbatlas_federated_database_instance_migration_test.go

```diff
@@ -1,158 +0,0 @@
-package mongodbatlas
```

Comment:
why did we have these tests if the resource is still using sdkv2? 🤔

Reply:
The resource isn't using SDKv2, there were some methods in the acceptance testing files which were left out which were still using SDKv2, that has been updated now with this PR.

Reply:
> The resource isn't using SDKv2
I am a bit confused here. Is the federated resource/data source using the new framework then? It does not seem to me but I may be missing something here. Looking at the import of the data source I see `/terraform-plugin-sdk/v2`